crates/api-desc/src/radio.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::*;
+
+pub(crate) fn new() -> Item {
+    let docs = docs! {
+        /// Radio operations.
+    };
+    let name = "radio".into();
```

Comment:
How general is the term radio? Does it cover other things than what this API is doing? If yes, then we should nest those functions below another module named according to what this API is doing within the radio realm.

For example, if we were to support Bluetooth, Zigbee, Wifi, or other such APIs, they should be at the same level as this API (which is suspect is something like BLE).